python/ray/util/sgd/torch/torch_trainer.py

```diff
@@ -96,20 +97,12 @@ def data_creator(config):
             TrainingOperator.
         config (dict): Custom configuration value to be passed to
             all creator and operator constructors.
-        dataloader_config (dict): Configuration values to be passed into
-            the ``torch.utils.data.DataLoader`` object that wraps
-            the dataset on each parallel worker for both training
-            and validation. Note that if ``num_replicas``
-            is greater than 1, ``shuffle`` and ``sampler`` will be
-            automatically set. See the available arguments
-            here https://pytorch.org/docs/stable/data.html.
-        num_replicas (int): the number of workers used in distributed
-            training.
+        num_workers (int): the number of workers used in distributed
```

Comment:
When I was testing the tensorflow SGD stuff, I had `num_workers=1` still wrap the worker in `DistributedDataParallel` for debug purposes and `num_workers=0` force local-only training. Maybe we should do the same here so that the "local" case is as close as possible to the distributed case for ease of debugging.

Reply:
Opened #7471 to track this.